packages/sdk/client-services/src/packlets/identity/identity-manager.ts

```diff
@@ -134,6 +136,16 @@ export class IdentityManager {
       // NOTE: This credential is written last. This is a hack to make sure that display name is set before identity is "ready".
       credentials.push(await generator.createDeviceAuthorization(identityRecord.deviceKey));
 
+      // Write device metadata to profile.
+      credentials.push(
+        await generator.createDeviceProfile({
+          platform: platform.name ?? '',
```

Comment:
you can set them to be optional in protobuf so undefined is would allowed

using empty strings is kinda ugly